interpreter.asm

```diff
@@ -375,27 +358,6 @@ FIND_NAME ; ( caddr u -- nt | 0 )
     and #STRLEN_MASK
     jmp .string_compare_failed
 
-GET_IMMED ; ( nt -- 1 | -1 )
-    lda MSB, x
-    sta W + 1
-    lda LSB, x
-    sta W
-    ldy #0
-
-    lda (W), y ; a contains string length + mask
-    and #F_IMMEDIATE
-    beq .not_immed
-    sty MSB, x ; 0
-    iny
-    sty LSB, x ; 1
-    rts
-
-.not_immed
-    lda #$ff
-    sta LSB, x
-    sta MSB, x
-    rts
-
     +BACKLINK ">xt", 3
```

Comment:
Used in the codebase as `latest >xt` for various self-reference.  This could possibly be factored to smaller forth code.